README.md

```diff
@@ -2,10 +2,18 @@
 
 <img align="right" src="docs/images/castle-logo.png">
 
+[![Join the chat at https://gitter.im/castleproject/all](https://badges.gitter.im/castleproject/all.svg)](https://gitter.im/castleproject/all?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
I think having this right at the top with "chat on gitter" might encourage people to seek out support questions there, we probably should add a "Contributing" header with badge and a link to our docs, a header and text like CONTRIBUTING.md would be good.